05/app.js

```diff
@@ -0,0 +1,109 @@
+const form = document.querySelector("form");
+const firstName = document.querySelector("input[name=firstName]");
+const lastName = document.querySelector("input[name=lastName]");
+const street = document.querySelector("input[name=street]");
+const houseNumer = document.querySelector("input[name=houseNumber]");
+const flatNumber = document.querySelector("input[name=flatNumber");
+const zip = document.querySelector("input[name=zip]");
+const city = document.querySelector("input[name=city]");
+const voivodeship = document.querySelector("select[name=voivodeship]");
+const submitBtn = document.querySelector("input[type=submit]");
+const ulEl = document.querySelector("ul");
+
+
+submitBtn.addEventListener('click', (e) => {
+    const errors = []
+  
+    if (firstName.value.length === 0) {
+      errors.push('Musisz wypelnic pole z imieniem');
+      changeBorderColor(firstName);
+    } else {
+        changeBackBorderColor(firstName);
```

Comment:
Można na samym końcu to zrobić przez zakolorowaniem na czerwono - wspominałem o tym w poprzednim zadaniu ;)